src/Components/QuestionComponentContainer/QuestionComponentContainer.css

```diff
@@ -1,3 +1,50 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@400;700&display=swap');
+
+/* h1,
+h3 {
+  font-family: 'Roboto Slab', serif;
+  font-weight: bold;
+  color: #b85a27;
+}
+
+h2 {
+  font-family: 'Roboto Slab', serif;
+  font-weight: bold;
+  color: #293457;
+}
+
+h4,
+h5,
+h6 {
+  font-family: 'Roboto Slab', serif;
+  font-weight: bold;
+  color: #293457;
+}
+*/
+
+p {
+  font-family: 'Open Sans', sans-serif;
+  color: #000000;
+}
+
+.question-header-container {
+  margin-bottom: 1.375rem;
+}
+
+.question-header {
+  font-family: 'Roboto Slab', serif;
+  font-weight: bold;
+  color: #b85a27;
+}
+
+.question-secondary-header {
+  font-size: 1.2rem;
+  font-family: 'Roboto Slab', serif;
+  font-weight: bold;
+  text-transform: uppercase;
+  color: #293457;
```

Comment:
Colors should be added to the `styleController.ts` file. More details in this [pr](https://github.com/Gary-Community-Ventures/benefits-calculator/pull/762/files#r1408155457).

Reply:
I've transitioned the colors.